app/onboarding/yay/page.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/no-unescaped-entities */
+import { Background } from '../styles';
+import {
+  Circle,
+  ContinueButton,
+  ContinueText,
+  InlineContainer,
+  ReviewContainer,
+  Title,
+} from './styles';
+
+export default function Onboarding() {
+  return (
+    <Background>
+      <Circle />
+      <InlineContainer>
+        <ReviewContainer>
+          <Title>You're all set!</Title>
+          <text>
+            Lorem ipsum dolor sit amet consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore.
+          </text>
+          <ContinueButton>
+            <ContinueText>Continue</ContinueText>
```

Comment:
import `SMALL` and replace with `<SMALL>Continue<SMALL>`. might need to put in extra props such as colors but not entirely sure